transport-config/transport-config.js

```diff
@@ -0,0 +1,142 @@
+
```

Comment:
awesome! you know this is worth making into a new standalone plugin :)
let's put that on the todo list